### PR TITLE
Cluster with load balancer and mirroring enabled

### PR DESCRIPTION
**Pull request by ramonsmits**

### CODE REVIEW

**Comment by WilliamBZA** [comment]
Is this something we want to document given Pivotal's stance on mirrored queues?

https://www.rabbitmq.com/ha.html#what-is-mirroring

> Important: mirroring of classic queues will be removed in a future version of RabbitMQ. Consider using quorum queues or a non-replicated classic queue instead.

**Comment by ramonsmits** [comment]
@WilliamBZA This just is the typical scenarios for all current customers on RabbitMQ. I had the need to setup a such a cluster. As we do not support quorum just yet I think this is very useful.

Added a bit of text about classic mirrored queues.

File: README.md [hunk 1]
```diff
@@ -34,4 +38,71 @@ docker exec rabbit3 rabbitmqctl join_cluster rabbit@rabbit1
...
+
+```cmd
+docker exec rabbit1 rabbitmqctl set_policy ha-all "" '{"ha-mode":"all","ha-sync-mode":"automatic"}'
+```
+
+Setup quorem queues:
+
+[quorum queues](https://www.rabbitmq.com/quorum-queues.html)
+
+
```
Comment (WilliamBZA) [inline README.md:53]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -34,4 +38,71 @@ docker exec rabbit3 rabbitmqctl join_cluster rabbit@rabbit1
...
+        server          rabbitmq1 rabbitmq1:15672  check inter 5s rise 2 fall 3
+        server          rabbitmq2 rabbitmq2:15672  check inter 5s rise 2 fall 3
+        server          rabbitmq3 rabbitmq3:15672  check inter 5s rise 2 fall 3
+```
+
+Setup HAProxy container, note correct the path where `haproxy.cfg` is saved.
+```cmd
+docker run -d --network rabbitnet --hostname rabbitha --name rabbitha -p 15672:15672 -p 5672:5672 -v ./haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg:ro haproxy:1.7
+```
+
```
Comment (WilliamBZA) [inline README.md:107]:
```suggestion

Setup quorem queues:

[quorum queues](https://www.rabbitmq.com/quorum-queues.html)

```

**Review by dvdstelt — APPROVED** [review]

**Pull request merged**